<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataExtractor</class>
 <widget class="QDialog" name="DataExtractor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>910</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>New Event Maker</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>870</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="mapB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Map</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitB">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="step2GB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>521</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { 
	border: 2px solid gray; 
	border-radius: 9px; 
	margin-top: 0.5em
}

QGroupBox::title {
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string>Step 2. Extract data</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="extractB">
       <property name="text">
        <string>Extract data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetB">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="durationED">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>120</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="displayFormat">
        <string>yyyy/MM/dd hh:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="timeLB">
       <property name="text">
        <string>Start time (GMT)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="durationLB">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="step1GB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>521</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { 
	border: 2px solid gray; 
	border-radius: 9px; 
	margin-top: 0.5em
}

QGroupBox::title {
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string>Step 1. Load stations Infomation</string>
   </property>
   <widget class="QTableWidget" name="staInfoTW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>501</width>
      <height>141</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>71</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <column>
     <property name="text">
      <string>Station</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Channel</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Location</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Network</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Latitude</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Longitude</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Elevation</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="loadStaInfoB">
       <property name="text">
        <string>Load a station file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewStaLocB">
       <property name="text">
        <string>View stations location</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="staFileNameLB">
       <property name="text">
        <string>File name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descLB">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="descLE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="staFileNameLE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="step3GB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>521</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { 
	border: 2px solid gray; 
	border-radius: 9px; 
	margin-top: 0.5em
}

QGroupBox::title {
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string>Step 3. Pick using Geotool</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="runGeoB">
       <property name="text">
        <string>View waveform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePickB">
       <property name="text">
        <string>Save picks</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="pickInfoTW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>151</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>125</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <column>
     <property name="text">
      <string>Station</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Channel</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="step4GB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { 
	border: 2px solid gray; 
	border-radius: 9px; 
	margin-top: 0.5em
}

QGroupBox::title {
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string>Step 4. Run Locator</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="editParamNLLocB">
       <property name="text">
        <string>Edit parameters for NLLoc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runNLLocB">
       <property name="text">
        <string>Run NLLoc</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="resultGB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>521</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { 
	border: 2px solid gray; 
	border-radius: 9px; 
	margin-top: 0.5em
}

QGroupBox::title {
	subcontrol-origin: margin;
	left: 10px;
	padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string>Result (Database Infomation)</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Algorithm</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>EVID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="timeLE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="evidLE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ORID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="oridLE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Latitude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Longitude</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="latLE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="lonLE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Depth (Km)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="depthLE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="algoLE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>staFileNameLE</tabstop>
  <tabstop>descLE</tabstop>
  <tabstop>loadStaInfoB</tabstop>
  <tabstop>viewStaLocB</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>durationED</tabstop>
  <tabstop>extractB</tabstop>
  <tabstop>resetB</tabstop>
  <tabstop>runGeoB</tabstop>
  <tabstop>savePickB</tabstop>
  <tabstop>editParamNLLocB</tabstop>
  <tabstop>runNLLocB</tabstop>
  <tabstop>saveB</tabstop>
  <tabstop>quitB</tabstop>
  <tabstop>staInfoTW</tabstop>
  <tabstop>pickInfoTW</tabstop>
  <tabstop>timeLE</tabstop>
  <tabstop>evidLE</tabstop>
  <tabstop>oridLE</tabstop>
  <tabstop>latLE</tabstop>
  <tabstop>lonLE</tabstop>
  <tabstop>depthLE</tabstop>
  <tabstop>algoLE</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
